### ISSUE TITLE
Blind accessibility: the buttons are not labeled, not usable with screen reader

### ISSUE THREAD

**Issue by 007vasy** [openai/openai-openapi#127]
Blind accessibility: the buttons are not labeled on the chatgpt app it makes it hard to blind person to use screen reader to use it

**Comment by logankilpatrick** [comment]
Hey @007vasy thank you for flagging this, I passed this along to the team so we can investigate further. I am going to close the issue since this repo is unrelated to ChatGPT but appreciate the feedback nonetheless!
